README.md

```diff
@@ -91,6 +91,10 @@ docker push <image_tag>
   * `<MSI_CLIENT_ID>` - This will use a specific client_id when authenticating with MSI. This is useful if you are using user-assigned identities.
   * `<MSI_OBJECT_ID>` - This will use a specific object_id when authenticating with MSI. This is useful if you are using user-assigned identities.
   * `<MSI_CLIENT_ID>` - This will use a specific MSI resource ID when authenticating with MSI. This is useful if you are using user-assigned identities.
+  * `<USE_ENV>` - if set to `true`, it will use service principal from envirorment variable for authenticating.
+  * `<TENANT_ID>` - Tenant ID of the service principal
+  * `<CLIENT_ID>` - Client ID of the service principal
+  * `<CLIENT_SECRET>` - Client Secret of the service principal
```

Comment:
My only concern is that you are going to pass the client secret via the environment variables of a pod.  This could expose the value outside of kubernetes secrets if this value is not defined in kubernetes as coming from a kubernetes secret.  This is only a documentation question.

Reply:
Although this feature is useful in case of ACI containers, there is a way to securely pass environment variable in Kubernetes as well: https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables. 
I will add this information in the document.

Reply:
Updated the document. Please review it

Reply:
Thank you for expanding the documentation.  This is what I was referring to when I said "defined in kubernetes as coming from a kubernetes secret" - thank you.